clr-k8s-examples/10-cri-resource-manager/clean.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
you should probably have a copyright header.
Shout if you cannot find an appropriately formatted one to copy from.

Reply:
done.